spec/functional/command/env_spec.rb

```diff
@@ -0,0 +1,32 @@
+require File.expand_path('../../../spec_helper', __FILE__)
```

Comment:
`env_spec.rb` was actually misplaced and was shipping with our sources. This moves it to its correct location.